discord/public_flags.py

```diff
@@ -0,0 +1,96 @@
+from .enums import UserFlags
```

Comment:
This is missing the license header present in every file, along with the encoding header.